config/samples/users/org2admin.yaml

```diff
@@ -7,26 +7,7 @@ metadata:
 spec:
   description: org2admin 用户信息的描述
   email: org2admin@tenxcloud.com
-  groups:
-  - observability
-  - system:nodes
-  - system:masters
-  - resource-reader
-  - iam.tenxcloud.com
-  - observability
   name: org2admin
-  password: $2a$10$693K.zP98yCs1qVwEp//DuWYOtLIE1doihtGhcCyYh3IpgSdGGba2
+  password: passw0rd
```

Comment:
The password should be encrypted.

Reply:
passw0rd -> $2a$10$eXHv4JD1hEfMA/vV6Whjae84bVI0CMeZpa43z97PurPQMmD8OAfe2